src/Behat/Testwork/Argument/MixedArgumentOrganiser.php

```diff
@@ -53,15 +55,40 @@ private function prepareArguments(array $parameters, array $arguments)
 
         list($named, $typehinted, $numbered) = $this->splitArguments($parameters, $arguments);
 
+        $someArgumentsAreKnown = $this->hasKnownArgument($numbered);
```

Comment:
I'm not sure about using "known" for these methods / variables. I might think that meant e.g. an argument that was "known" because it matches a variable name or typehint where really these are the opposite of that.

Cucumber tends to call tables and strings ["multiline step arguments"](https://cucumber.io/docs/gherkin/reference/#examples) and Behat also uses `multiline` to mean tables and strings for things like [the formatter option for whether to display them in full](https://github.com/Behat/Behat/blob/master/src/Behat/Config/Formatter/PrettyFormatter.php#L24).

So maybe `$wasMultilineProvided = $this->hasMultilineArgument($numbered)` and `if ($wasMultilineProvided && ! $this->hasMultilineArgument($arguments))`?

Reply:
Oh I know, sometimes I wish instead of an AI that completed the code based on my variable/method names, I could have one where I write the code and the AI just gives everything a good name 🤣